hotsos/core/host_helpers/cli.py

```diff
@@ -395,7 +396,12 @@ def format_json_contents(self, *args, **kwargs):  # pylint: disable=W0613
         with open(self.path) as f:
             lines = f.readlines()
 
-        if lines and lines[-1].startswith(self.last_line_filter):
+        if self.first_line_filter:
+            line_filter = self.first_line_filter
```

Comment:
Will it be simple to use python inbuilt function string strip() to handle this?
strip() can remove both leading and trailing characters.

Reply:
I am not sure I understand.

Do you mean loading the entire file into a string and then removing the parts from either end and writing that string back?

What we try to do here is remove the first or last line which looks like in either form:
```
report xyz
{
 # json formatted output
}
```
or 
```
{
 # json formatted output
}
report xyz
```

The extra line "report xyz" makes invalid json and parsing fails. So removing that extra line makes it work.